test/specs/index.js

```diff
@@ -16,43 +16,34 @@ describe('lightbox', function() {
     setTimeout(fn, 2 * FADE_TIME + 16);
   }
 
-  // must close it before running another test as there is no destructor
-  function close(done) {
-    tempWait(() => {
-      $(CLOSE_CLASS).click();
-      tempWait(() => {
-        expect($(BLOCKING_CLASS)).not.toBeVisible();
-        done();
-      });
-    });
-  }
-
   beforeEach(function() {
     const lightboxDiv = (id) => `${LIGHTBOX_CLASS}[data-src="${imagePath(id)}"] img[style="width:50px;height:50px"]`;
     affix(`${lightboxDiv(1)}+${lightboxDiv(2)}`);
+
+    this.init = (...args) => this.unit = lightbox.init.apply(lightbox, args);
   });
 
-  afterEach(function(done) {
-    close(done);
+  afterEach(function() {
+    this.unit.destroy();
   });
 
   it('should render the blocking div when clicked on the lightbox node', function() {
-    lightbox.init();
-    expect($(BLOCKING_CLASS)).not.toBeInDOM();
+    this.init();
+    expect($(BLOCKING_CLASS).is(':off-screen')).toBeTruthy();
```

Comment:
We want the slide to be fully rendered (with images, videos, iframes) off-screen (i.e., `transform: translate(-99999px);`). This detects that it is indeed not visible to the user.